src/wallet/salvage.cpp

```diff
@@ -6,10 +6,23 @@
 #include <fs.h>
 #include <streams.h>
 #include <util/translation.h>
+#ifdef USE_BDB
+#include <wallet/bdb.h>
+#endif
 #include <wallet/salvage.h>
 #include <wallet/wallet.h>
 #include <wallet/walletdb.h>
 
+#ifndef USE_BDB
+// If BDB is not compiled, we don't have a salvage
```

Comment:
Prefer to not compile the file at all...

Reply:
Done